src/app/components/display-request-page/display-request-page.component.ts

```diff
@@ -93,6 +95,24 @@ export class DisplayRequestPageComponent implements OnInit {
   }
   }
 
+  //Toggles the select format strings
```

Comment:
What happened to the alignment in this section? This code block should be tabbed if it is inside the component.

Reply:
Change this comment to a JSDoc comment.

Reply:
Changed.